apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -316,6 +384,18 @@ export default class LibraryPublisher extends React.Component {
         <Heading2>{i18n.description()}</Heading2>
         {this.displayDescription()}
         <Heading2>{i18n.catProcedures()}</Heading2>
+        <div style={styles.functionSelector}>
+          <input
+            style={styles.largerCheckbox}
+            type="checkbox"
+            id="func-select-all"
```

Comment:
Less critical here, but would also be nice to use lodash unique IDs - especially as we are starting to use this pattern more.